tests/test-object-cache.php

```diff
@@ -509,8 +509,8 @@ public function test_get_multi_returns_array_of_values_from_memcache(): void {
 		);
 
 		$expected = [
-			$this->object_cache->key( 'foo', 'default') => 'data',
-			$this->object_cache->key( 'foo', 'another-group') => 'data',
+			'default:foo' => 'data',
+			'another-group:foo' => 'data',
```

Comment:
For the moment I've just done the basic adaptations to the previous tests to validate that everything works the same apart from the slight response format. I'll add more tests (directly to `wp_cache_get_multiple()`) later